### ISSUE TITLE
Host AICA C++ style guide

### ISSUE THREAD

**Issue by domire8** [aica-technology/control-libraries#94]
The style guide link in the [contributing guide](https://github.com/aica-technology/control-libraries/blob/main/CONTRIBUTING.md#style-guide) points to a private EPFL wiki. We need a find a place to host the style guide ourselves for 7.0.

@eeberhard I allow myself to assign this to you, I think I changed my style config in a few places at some point, so it's not the original one.

**Comment by eeberhard** [comment]
Yes agreed, I already have #49 and need to update the contribution guide after #27 anyway, so this is a good related issue.
